src/views/product-tutorials-view/components/product-view-content/components/branded-callout/types.ts

```diff
@@ -2,11 +2,6 @@ import { ProductOption } from 'lib/learn-client/types'
 
 export interface BrandedCalloutProps {
   heading: string
-  /**
-   * Identifier for the heading, which should unique in the context of the page
-   * Note: headingSlug is added after fetching content from the Learn API
-   */
-  headingSlug: string
```

Comment:
`BrandedCallout` no longer renders a heading or appears in the table-of-contents, so it has no need for `headingSlug`.